jormungandr/src/explorer/graphql/scalars.rs

```diff
@@ -33,6 +36,28 @@ pub struct PublicKey(pub String);
 #[derive(juniper::GraphQLScalarValue)]
 pub struct TimeOffsetSeconds(pub String);
 
+// u32 should be enough to count blocks and transactions (the only two cases for now)
+#[derive(Clone)]
+pub struct IndexCursor(pub u32);
+
+juniper::graphql_scalar!(IndexCursor where Scalar = <S> {
+    description: ""
```

Comment:
Hmm, we ought to have some nice GraphQL metadata to our API if we can get it for free here.

Reply:
You are right